central/graphql/resolvers/gen/main.go

```diff
@@ -105,6 +107,14 @@ var (
 				ParentType: reflect.TypeOf(storage.CVE{}),
 				FieldName:  "CvssV3",
 			},
+			{
+				ParentType: reflect.TypeOf(storage.ImageCVEV2{}),
```

Comment:
Why skip these?

Reply:
That's a good question.  I can't remember why i did that when hacking around.  I will unskip them.  Can always add it back it in if subsequent PR proves we need to.